aws/alarms/cloudwatch.tf

```diff
@@ -99,6 +99,24 @@ resource "aws_cloudwatch_log_metric_filter" "application_error" {
   }
 }
 
+resource "aws_cloudwatch_metric_alarm" "5xx_error_warn" {
+  alarm_name          = "HTTPCode_ELB_5XX_Count"
+  alarm_description   = "ELB Warning - 5xx Error exceed threshold."  
+  comparison_operator = "GreaterThanThreshold"
+  evaluation_periods  = "1"
+  metric_name         = "HTTPCode_ELB_5XX_Count"
+  namespace           = "forms"
+  period              = "300"
+  statistic           = "Sum"
+  threshold           = "10"
```

Comment:
Wouldn't we want to know if there is a single 500 error ? That generally means something is going really wrong with our services

Reply:
I wasn't sure what the threshold should be, so just grabbed the same config from Notify. It can certainly be lowered.